nginx/nginx.example.conf

```diff
@@ -44,10 +46,10 @@ http {
 			proxy_cache_bypass $http_upgrade;
 		}
 
-		ssl_protocols TLSv1 TLSv1.1;
+		ssl_protocols TLSv1 TLSv1.1 TLSv1.2 TLSv1.3;
```

Comment:
Is there a reason these were added? They aren't used by the console, they won't get used

Reply:
The main reason is simply that I didn't see any reason to exclude newer protocols.

Reply:
Well, the reason being that they're completely unused. The only clients that will ever connect here are incapable of using these protocols, so there's no reason to have them in the first place